frontend/www/js/omegaup/components/problem/Details.vue

```diff
@@ -211,7 +218,7 @@ import problem_Feedback from './Feedback.vue';
 import problem_SettingsSummary from './SettingsSummaryV2.vue';
 import problem_Solution from './Solution.vue';
 import qualitynomination_Demotion from '../qualitynomination/DemotionPopup.vue';
-import qualitynomination_Promotion from '../qualitynomination/Popup.vue';
+import qualitynomination_Promotion from '../qualitynomination/PromotionPopup.vue';
```

Comment:
```suggestion
import qualitynomination_PromotionPopup from '../qualitynomination/PromotionPopup.vue';
```

igual los demás popups para que sean consistentes con cómo se importan.

Reply:
Listo